metaflow/plugins/retry_decorator.py

```diff
@@ -22,10 +38,14 @@ class RetryDecorator(StepDecorator):
         Number of times to retry this task.
     minutes_between_retries : int, default 2
         Number of minutes between retries.
+    only_on : List[str], default None
+        List of failure events to retry on. Not specifying values will retry on all known events.
+        Accepted values are
+        'step', 'instance-preemption'
```

Comment:
this can be `all`, `instance-preemption`. `all` can be the default value and is the same as the current behavior.

Reply:
but do we even need the `all` as a value, as that behaviour should be covered by omitting the attribute completely? is it also not a plausible use-case to retry only on user-code exceptions?